### PR TITLE
update sql clauses to 3.10.0

### PR DESCRIPTION
**Pull request by kunkun-tang**
Multi Executor mode is already applied in production for more than 1 years. It's time to update the sql clauses to match the latest multi executor Mode.

### CODE REVIEW

**Comment by HappyRay** [comment]
How did you test?

**Comment by HappyRay** [comment]
@jamiesjc could you review this change?

**Comment by fsi206914** [comment]
I already released the new sql changes to one of our new AZ DB. Will evaluate how it behave after AZ is set up there. @HappyRay

File: azkaban-sql/src/sql/database.properties [hunk 1]
```diff
@@ -1 +1 @@
-version=3.0
+version=3.10.0
```
Comment (fsi206914) [inline azkaban-sql/src/sql/database.properties]: I've no much idea how to modify this version. Just give it a recent number.

Comment (suvodeep-pyne) [inline azkaban-sql/src/sql/database.properties]: I don't think this file is used anywhere. I'm ok with deleting this file.

**Comment by suvodeep-pyne** [comment]
Hey @fsi206914, How are the sql files being used / read in the code ?

**Comment by fsi206914** [comment]
@suvodeep-pyne I believe these sql files were not read/used in the internal AZ code. I just copy tables one by one every time I set up an AZ db.

**Comment by fsi206914** [comment]
removed database.properties file.

**Comment by HappyRay** [comment]
The file database.properties is referenced in  azkaban.database.AzkabanDatabaseSetup#findOutOfDateTable.

**Comment by kunkun-tang** [comment]
@HappyRay I checked the logic. There is a boolean configuration database.check.version. If it is true, we will do the setup table automatically when web server start. I believe we should keep safe and get the file back to repo.

**Comment by kunkun-tang** [comment]
Evaluated in our another az cluster, and it is working well.

**Pull request merged**